src/v1/types/responses/service_info.rs

```diff
@@ -65,32 +65,39 @@ pub struct ServiceInfo {
     ty: ServiceType,
 
     /// An optional description of the service.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     description: Option<String>,
 
     /// The organization running the service.
     organization: Organization,
 
     /// An optional contact URL.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     contact_url: Option<String>,
 
     /// An optional documentation URL.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     documentation_url: Option<Url>,
 
     /// Timestamp when the service was first available.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     created_at: Option<DateTime<Utc>>,
 
     /// Timestamp when the service was last updated.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     updated_at: Option<DateTime<Utc>>,
 
     /// An optional string describing the environment that the service is
     /// running within.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     environment: Option<String>,
 
     /// The version of the service.
     version: String,
 
     /// Lists some, but not necessarily all, storage locations supported by the
     /// service.
+    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
     storage: Option<Vec<String>>,
```

Comment:
Is this missing `tesResources_backend_parameters` (see the entry in https://editor.swagger.io/?url=https://raw.githubusercontent.com/ga4gh/task-execution-schemas/refs/heads/main/openapi/task_execution_service.openapi.yaml)